content/blog/posts/devops-daycamp-2015.rst

```diff
@@ -0,0 +1,38 @@
+DevOps DayCamp
```

Comment:
Perhaps we might want to change this to add ``2015`` at the end to distinguish between the previous daycamp post we did last year?